slack_bolt/function/function.py

```diff
@@ -0,0 +1,86 @@
+from typing import List, Union, Pattern, Callable, Dict, Optional, Sequence
+from logging import Logger
+
+from slack_bolt.listener_matcher import builtins as builtin_matchers
+
+from slack_bolt.response import BoltResponse
+from slack_bolt.middleware import Middleware
+from slack_bolt.middleware.function_token import FunctionToken
+
+from slack_bolt.util.utils import to_listener_functions
+
+
+class Function:
```

Comment:
If we don't support the view method for now, can you add TODO comment about it?

Reply:
I was going to add this in a different PR, But will add this as a TODO